# Thomas Hoffmann, EMBL Heidelberg, structures-it@embl.de, 2023/11
# mdocfile requires pydantic > 2.0 due to "field_validator" and "GenericModel".
# This conflicts with the requirement of napari 4.18 and morphosamplers 0.0.9, both used by RELION 5.
# use rename py module pydantic2 
diff -ru mdocfile-0.1.0/PKG-INFO mdocfile-0.1.0_use_pydantic2/PKG-INFO
--- mdocfile-0.1.0/PKG-INFO	2020-02-02 01:00:00.000000000 +0100
+++ mdocfile-0.1.0_use_pydantic2/PKG-INFO	2023-11-09 16:26:43.336778039 +0100
@@ -16,7 +16,7 @@
 Classifier: Programming Language :: Python :: 3.11
 Requires-Python: >=3.9
 Requires-Dist: pandas
-Requires-Dist: pydantic>=2
+Requires-Dist: pydantic2>=2
 Provides-Extra: dev
 Requires-Dist: black; extra == 'dev'
 Requires-Dist: ipython; extra == 'dev'
diff -ru mdocfile-0.1.0/pyproject.toml mdocfile-0.1.0_use_pydantic2/pyproject.toml
--- mdocfile-0.1.0/pyproject.toml	2020-02-02 01:00:00.000000000 +0100
+++ mdocfile-0.1.0_use_pydantic2/pyproject.toml	2023-11-09 16:27:01.832239370 +0100
@@ -25,7 +25,7 @@
 dynamic = ["version"]
 dependencies = [
     "pandas",
-    "pydantic>=2"
+    "pydantic2>=2"
 ]
 
 # extras
diff -ru mdocfile-0.1.0/src/mdocfile/data_models.py mdocfile-0.1.0_use_pydantic2/src/mdocfile/data_models.py
--- mdocfile-0.1.0/src/mdocfile/data_models.py	2020-02-02 01:00:00.000000000 +0100
+++ mdocfile-0.1.0_use_pydantic2/src/mdocfile/data_models.py	2023-11-09 16:27:17.368628936 +0100
@@ -1,4 +1,4 @@
-from pydantic import field_validator, BaseModel
+from pydantic2 import field_validator, BaseModel
 from pathlib import Path
 from typing import List, Optional, Tuple, Union, Sequence
 
